commands/profile.py

```diff
@@ -24,7 +24,7 @@ async def profile(self, ctx, user : discord.Member = None):
         else:
             em = discord.Embed(title=user.name, colour=eval(profile[4]))

```

Comment:
Why is there even an eval call here?

Reply:
This is out of scope for this PR. Please raise an issue instead.